### PR TITLE
ci: add devmapper test for dragonball

### PR DESCRIPTION
**Pull request by Tim-0731-Hzt**

### CODE REVIEW

File: tests/integration/kubernetes/gha-run.sh [hunk 1]
```diff
@@ -173,13 +173,7 @@ function run_tests() {
 
 	pushd "${kubernetes_dir}"
 	bash setup.sh
-	if [[ "${KATA_HYPERVISOR}" = "dragonball" ]] && [[ "${SNAPSHOTTER}" = "devmapper" ]]; then
-		echo "Skipping tests for dragonball using devmapper"
-	elif [[ "${KATA_HYPERVISOR}" = "cloud-hypervisor" ]]; then
-		echo "Skipping tests for ${KATA_HYPERVISOR}"
```
Comment (justxuewei) [inline tests/integration/kubernetes/gha-run.sh:179]: Run this test even if hypervisor is clh?

Comment (Tim-0731-Hzt) [reply tests/integration/kubernetes/gha-run.sh]: fixed

**Review by fidencio — APPROVED** [review]
lgtm, thanks @Tim-0731-Hzt!

**Pull request closed without merging**